aries-site/src/examples/components/controls/MenuExample.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Box } from 'grommet';
+import { Menu } from 'aries-core';
+
+export const MenuExample = () => {
+  const items = [
+    { label: 'Past 12 months' },
+    { label: 'Past 6 months' },
+    { label: 'Past 3 months' },
+    { label: 'Past 30 days' },
+    { label: 'Past 7 days' },
+  ];
+  return (
+    <Box background="light-2" round="xsmall">
```

Comment:
Should we try to stick to using colors in the aries theme? If so, `background-front` is the color I believe we should be using.

Reply:
Also, going through this is making me wonder if these examples should be shown in both light and dark contexts? Mocks only show one context for controls, but do we want to be consistent with branding.js and color.js?

Reply:
You're right. We should stick with the aries theme colors. I think it also should support light and dark contexts. Let me make adjustments to ensure that's the case.

Reply:
The reason I had put a light background on that box was so that the Menu thought it was in a light context and would render light theme colors for the button -- otherwise, the control button blends in with the page background because there is no border. I still think the example should support light and dark modes, but I'm wondering how to better distinguish the Menu button from the background if it's showing dark mode menu in dark mode site. @halocline